infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -86,32 +103,61 @@ versioned_builds = [
     git_tag         = "v2.0.0"
     package_version = "2.0"
     accelerator     = "tpu"
+    python_version  = "3.8"
   },
   {
     git_tag         = "v1.13.0"
     package_version = "1.13"
     accelerator     = "tpu"
+    python_version  = "3.8"
+  },
+  {
+    git_tag         = "v2.1.0"
+    pytorch_git_rev = "v2.1.0"
+    package_version = "2.1.0"
+    accelerator     = "cuda"
+    cuda_version    = "11.8"
+    python_version  = "3.8"
   },
   {
     git_tag         = "v2.1.0"
     pytorch_git_rev = "v2.1.0"
     package_version = "2.1.0",
     accelerator     = "cuda"
     cuda_version    = "12.0"
+    python_version  = "3.8"
+  },
+  {
+    git_tag         = "v2.1.0"
+    pytorch_git_rev = "v2.1.0"
+    package_version = "2.1.0"
+    accelerator     = "cuda"
```

Comment:
I think we only need the CUDA 12.1 build, right? That's the one that will work with the default torch wheel IIRC